src/View/Widget/MultiCheckboxWidget.php

```diff
@@ -112,7 +112,8 @@ public function render(array $data, ContextInterface $context)
             'disabled' => null,
             'val' => null,
             'idPrefix' => null,
-            'templateVars' => []
+            'templateVars' => [],
+            'label' => true
```

Comment:
It has to remain as `label` in `RadioWidget` as it can be called direct from `FormHelper::radio`. Also you could call `FormHelper::multiCheckbox with `label` and it would work. I think it needs to remain as `label` in the widgets for these direct method functionality too.